CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v4.3.0 (2020-12-16)
+
+## New features
+* Add prompt param support in login function that could be `select_account` or `login`
+* Prepare callback function for `showSimplifiedLoginWidget` that fire login with `promp=login`
```

Comment:
Can we just have one point,
* Add `not you?` button to simplified login widget
?
Since from clients perspective nothing has change in `login` or `showSimplifiedLoginWidget`. That `prompt` is not intended to be change by clients